modules/virt-about-upgrading-virt.adoc

```diff
@@ -5,20 +5,23 @@
 [id="virt-about-upgrading-virt_{context}"]
 = About upgrading {VirtProductName}
 
+[id="how-upgrades-work_{context}"]
 == How {VirtProductName} upgrades work
 
-* You can upgrade to the next minor version of {VirtProductName} by using the
-{product-title} web console to change the channel of your Operator subscription.
+* Operator Lifecycle Manager (OLM) manages the lifecycle of the {VirtProductName} Operator. The Marketplace Operator, which is deployed during {product-title} installation, makes external Operators available to your cluster.
 
-* You can enable automatic _z-stream_ updates during {VirtProductName} installation.
+* OLM provides z-stream and minor version updates for {VirtProductName}. Minor version updates become available when you upgrade {product-title} to the next minor version. You cannot upgrade {VirtProductName} to the next minor version without first upgrading {product-title}.
 
-* Updates are delivered via the _Marketplace Operator_, which is deployed
-during {product-title} installation. The Marketplace Operator makes
-external Operators available to your cluster.
+* {VirtProductName} subscriptions use a single update channel that is named *stable*. The *stable* channel ensures that your {VirtProductName} and {product-title} versions are compatible.
+
+** If your subscription's approval strategy is set to *Automatic*, the upgrade process starts as soon as a new version of the Operator is available in the *stable* channel. If the approval strategy is set to *Manual*, you must manually approve pending upgrades.
+
+** It is highly recommended to use the *Automatic* approval strategy to maintain a supportable environment. Each minor version of {VirtProductName} is only supported if you run the corresponding {product-title} version. For example, you must run {VirtProductName} {VirtVersion} on {product-title} {VirtVersion}.
```

Comment:
Can these two paragraphs be restructured to include "automatic" and "recommended" in the first paragraph.

The second should include "manual" and "not recommended", and "risk to support and functionality if getting out of sync with OCP"

Reply:
Thanks for the feedback! I'll do that now.